src/app/api/file/route.schema.ts

```diff
@@ -0,0 +1,51 @@
+import { z } from "zod";
+import { Prisma } from "@prisma/client";
+import { unauthorizedErrorSchema, unknownErrorSchema } from "@api/route.schema";
+/*
+  id           String   @id @default(auto()) @map("_id") @db.ObjectId
+  date         DateTime // will restrict hours to midnight 
+  filetype     String
+  url          String
+  seniorId     String   @db.ObjectId
+  senior       Senior   @relation(fields: [seniorId], references: [id], onDelete: Cascade)
+  Tags         String[]
+
+  @@unique([seniorId, date])
+*/
+
+export const File = z.object({
+  date: z.string(),
+  filetype: z.string(),
+  url: z.string(),
+  Tags: z.array(z.string()),
+  seniorId: z.string(),
+});
+
+export const FileResponse = z.discriminatedUnion("code", [
+  z.object({
+    code: z.literal("SUCCESS"),
+    message: z.literal("File successfully added"),
+  }),
+  z.object({
+    code: z.literal("INVALID_FILE"),
+    message: z.literal("Invalid file added"),
+  }),
+  z.object({
+    code: z.literal("UNKNOWN"),
+    message: z.literal("Unknown error received"),
+  }),
+  z.object({
```

Comment:
Maybe we can collapse `DUPLICATE_DATE` to `INVALID_FILE` since which dates the user can pick is limited on the frontend.